tests/js/server/aql/aql-queries-optimizer-in-noncluster.js

```diff
@@ -28,49 +28,36 @@
 /// @author Copyright 2012, triAGENS GmbH, Cologne, Germany
 ////////////////////////////////////////////////////////////////////////////////
 
-var jsunity = require("jsunity");
-var internal = require("internal");
-var helper = require("@arangodb/aql-helper");
-var getQueryResults = helper.getQueryResults;
-var db = internal.db;
+const jsunity = require("jsunity");
+const internal = require("internal");
+const helper = require("@arangodb/aql-helper");
+const getQueryResults = helper.getQueryResults;
+const db = internal.db;
 
 ////////////////////////////////////////////////////////////////////////////////
 /// @brief test suite
 ////////////////////////////////////////////////////////////////////////////////
 
 function ahuacatlQueryOptimizerInTestSuite () {
-  var c = null;
-  var cn = "UnitTestsAhuacatlOptimizerIn";
+  const cn = "UnitTestsAhuacatlOptimizerIn";
```

Comment:
Ahuacatl was removed in other parts of this diff?

Reply:
It's just a name. It doesn't really matter IMO.